user/enterprise/focal.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Focal Build Containers for Enterprise (beta)
+layout: en_enterprise
+
+---
+
+## System Setup
+
+**Platform Requirements**: To use the Focal build containers, the Travis CI Enterprise (TCIE) installation must be at version 3.0 or higher. 
+Please be sure to [install TCIE 3.x](/user/enterprise/tcie-3.x-setting-up-travis-ci-enterprise), if needed, before getting started.
+
+> It is possible to install Focal for Travis CI 2.2.6+, however Travis CI Enterprise Support Team will be not accepting any Focal build environment reports for TCIE version lower than 3.0.
```

Comment:
Gotcha. To be addressed with additional remark.

Reply:
After re-checking - focal environment compatibility with 2.2.x is not fully verified and there's a large codebase difference between TCIE 2.x and 3.x. I will remove this remark and leave explicit requirement of Travis CI 3.0 for that worker to avoid confusion and give a clear guideline. If anyone goes down that path, Travis team support may be needed anyway.